src/App.css

```diff
@@ -0,0 +1,42 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap");
```

Comment:
- [ ] Great mobile menu and implementation!  However:

![image](https://user-images.githubusercontent.com/57421823/231302286-23e984c2-f146-42db-ad7b-3c3325444ebd.png)

When your users select a link in the mobile menu, make it disappear so they don't have to press "X" first to see the requested section.

That way, the UX will be vastly improved on mobile. 💪 

_This was tested on both Chrome and Firefox._